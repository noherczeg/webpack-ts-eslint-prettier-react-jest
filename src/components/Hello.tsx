import * as React from "react";

export interface HelloProps {
  compiler: string;
  framework: string;
}

export const Hello = (props: HelloProps): JSX.Element => {
  const { compiler, framework } = props;

  return (
    <h1>
      Hello from {compiler} and {framework}!
    </h1>
  );
};
