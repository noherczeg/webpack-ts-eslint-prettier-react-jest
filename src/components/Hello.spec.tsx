import * as React from "react";
import * as renderer from "react-test-renderer";
import { Hello } from "./Hello";

describe("Hello", () => {
  test("snapshot renders", () => {
    const component = renderer.create(
      <Hello compiler="TypeScript" framework="React" />
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
