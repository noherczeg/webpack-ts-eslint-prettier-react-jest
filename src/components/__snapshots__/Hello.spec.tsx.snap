// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello snapshot renders 1`] = `
<h1>
  Hello from 
  TypeScript
   and 
  React
  !
</h1>
`;
